src/theme/index.ts

```diff
@@ -98,6 +102,7 @@ export {
   screens,
   darkTheme,
   extendTheme,
+  SENTIMENTS,
```

Comment:
really really not a fan, we try to use theme through the context all around, we should not do this :)

Reply:
@chambo-e what's the best way? I define sentiments directly in each components by using `Object.keys(theme.colors)`?

Reply:
@chambo-e what's the best way? I define sentiments directly in each components by using `Object.keys(theme.colors)`?